paddle/platform/profiler.cc

```diff
@@ -169,5 +172,149 @@ std::vector<std::vector<Event>> DisableProfiler() {
   return result;
 }
 
+void PushEvent(const std::string& name, DeviceContext* dev_ctx) {
+  GetEventList().Record(EventKind::kPushRange, std::move(name), g_thread_id,
+                        dev_ctx);
+}
+
+void PopEvent(const std::string& name, DeviceContext* dev_ctx) {
+  GetEventList().Record(EventKind::kPopRange, std::move(name), g_thread_id,
+                        dev_ctx);
+}
```

Comment:
Need to refine `RecordEvent::RecordEvent()` and `RecordEvent::~RecordEvent()` to reuse these two functions.

Reply:
Done